pom.xml

```diff
@@ -398,25 +407,40 @@
     </dependencyManagement>
 
     <dependencies>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <version>5.10.2</version>
```

Comment:
Any reason this couldn't use the variable `jupiter.version`?

Reply:
Similar question for other spots that have 5.10.2 like line 402 above.

Reply:
This is not a `jupiter` dependency and if we want to change the `jupiter` version, we don't know if it will apply here.

Reply:
OK if they are not related and maintained separately then that's ok.

Reply:
But we can have another variable for `junit` if need be.

Reply:
i dont think we need variables for junit related dependencies, those versions are different.